### ISSUE TITLE
Failed to start job File Crawler

### ISSUE THREAD

**Issue by PakanAngel** [codelibs/fess#1399]
I did configure a Fess file crawler on Centos 7 that didn't work at all. All files are in PDF format and are in a folder at root: /elasticsearch/repo but the crawler cannot find the folder. After changing some setups on the crawler now when I want to start the job, get the error:  Failed to start job File Crawler.
1- Why do I get this message and cannot start the job?
2- What is wrong in my crawler setting that it refuses to index those files?

File Crawler Settings: 
Name: Docs
Paths: file:/elasticsearch/repo
Depth: 3
Max Access Count: 1000
The Number of Thread: 1
Interval Time: 1000ms
Boost: 1.0
Permissions: {role}guest
Label: Docs
Status: Enabled

**Comment by marevol** [comment]
Did you check log files?
Is the directory permission correct?

**Comment by PakanAngel** [comment]
Here are 2 pictures showing the crawler.log for file crawling and the permission of the folder: /elasticsearch
Pictures:
![screen shot 1396-10-04 at 23 37 54 1](https://user-images.githubusercontent.com/26673019/34343415-103ffc62-e9ce-11e7-86ff-690dbdfc9de6.png)
![screen shot 1396-10-04 at 23 45 56](https://user-images.githubusercontent.com/26673019/34343416-1094debc-e9ce-11e7-85ab-d1eff54c5f10.png)

**Comment by PakanAngel** [comment]
And this shows the error of the file crawler in URL Failure page.

![screen shot 1396-10-05 at 00 17 07](https://user-images.githubusercontent.com/26673019/34343521-39c686e2-e9d2-11e7-8d13-84eca875e20b.png)

**Comment by PakanAngel** [comment]
I've also checked different file path syntaxes an below:
![screen shot 1396-10-05 at 12 31 07](https://user-images.githubusercontent.com/26673019/34355878-f608294e-ea38-11e7-9447-86184ddbd6a5.png)

**Comment by marevol** [comment]
Does Fess process run by fess user? If yes, /elasticsearch/repo and /elasticsearch should be accessible by fess user.

**Comment by PakanAngel** [comment]
I think fess is running under a user called "fess".

**Comment by PakanAngel** [comment]
I've changed the owner and permission of that folder to "fess" user, but still the same problem.
Can you check my crawler.log file to see what could cause this issue?
[fess-crawler.log](https://github.com/codelibs/fess/files/1587413/fess-crawler.log)

**Comment by marevol** [comment]
2017-12-26 14:48:17,225 [Crawler-AWCO2Fvrr5lvujcK90kV-1-1] INFO  Could not access file://elasticsearch/repo/

I do not think the permission is correct...

**Comment by PakanAngel** [comment]
The parent folder which is called "elasticsearch" has also the same permissions
![screen shot 1396-10-05 at 15 05 59](https://user-images.githubusercontent.com/26673019/34358248-66b5ee5a-ea4e-11e7-99a0-62ccf50bbbf5.png)

**Comment by PakanAngel** [comment]
I can also change my repo folder to another folder which Fess can access by default but which folder would be for this case?

**Comment by PakanAngel** [comment]
Finally solved. But don't really what caused the problem. I changed my file crawler folder to /usr/share/fess/bin which is the main Fess  folder. I realized that the crawler works. After a while I changed the crawl path to my initial repo folder at /elasticsearch/repo/ and suddenly it crawled all files residing there. But it is definite that you any folder you want to use as a repository for file crawlng should be set to fess user.
